Formula/s/swctl.rb

```diff
@@ -0,0 +1,26 @@
+class Swctl < Formula
+  desc "Apache SkyWalking CLI (Command-line Interface)"
+  homepage "https://skywalking.apache.org/"
+  url "https://github.com/apache/skywalking-cli/archive/refs/tags/0.14.0.tar.gz"
+  sha256 "9b1861a659e563d2ba7284ac19f3ae72649f08ac7ff7064aee928a7df2cd2bff"
+  license "Apache-2.0"
+  head "https://github.com/apache/skywalking-cli.git", branch: "master"
+
+  livecheck do
+    url :stable
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags:), "./cmd/swctl"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/swctl --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate teadd-gurkath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thanks for the reminder, added.